Engines/Wine/Plugins/virtual desktop/script.js

```diff
@@ -1,13 +1,23 @@
 include("engines.wine.engine.object");
+include("utils.functions.system.virtual_desktop");
 
 /**
  * sets Virtual Desktop with window resolution
  * @param {number} width width of virtual desktop (in px)
  * @param {number} height height of virtual desktop (in px)
  * @returns {Wine} Wine object
  */
+// eslint-disable-next-line no-unused-vars
```

Comment:
Is this annotation really necessary for `Wine.prototype.setVirtualDesktop`?

Reply:
No and it dosen't help with that prototype function causing Codacy error.